deepfence_frontend/apps/dashboard/src/features/integrations/components/integration-form/utils.ts

```diff
@@ -90,7 +90,10 @@ export const isJiraIntegration = (integrationType: string) => {
 };
 
 export const isCloudComplianceNotification = (notificationType: string) => {
-  return notificationType && notificationType === 'CloudCompliance';
+  return (
```

Comment:
can you double check? this condition should be like `notificationType && (notificationType === 'CloudCompliance' || notificationType.toLowerCase() === 'cloud_compliance')`